pulp_container/app/viewsets.py

```diff
@@ -100,6 +100,20 @@ class Meta:
         }
 
 
+class ManifestSignatureFilter(ContentFilter):
+    """
+    FilterSet for image signatures.
+    """
+
```

Comment:
it might be worth adding a filter for the signed manifest digest in a similar manner it's done here https://github.com/pulp/pulp_container/blob/main/pulp_container/app/viewsets.py#L66